src/components/ListItemCard.tsx

```diff
@@ -19,23 +26,39 @@ export default function ListItemCard({ className }: any) {
         className="rounded-lg"
       />
 
-      <div className="flex flex-col w-full pl-4 gap-4">
+      <div className="flex flex-col w-full gap-4">
         <div className="flex w-full justify-between">
-          <div className='flex flex-col gap-1'>
-            <span className="text-lg font-normal">Camasa motor Ford 35-121</span>
-            <span className="text-xl font-semibold">
-              177.00 RON
+          <div className="flex flex-col gap-1">
+            <span className="text-lg font-normal">
+              Camasa motor Ford 35-121
             </span>
+            <span className="text-xl font-semibold">177.00 RON</span>
           </div>
-          <ItemAddedPreviewModal/>
+          <Button text="Adauga" onClick={onAddToCart}>
```

Comment:
Related to the topic of controlling modals, from a larger parent component, and decoupling the modal from the card